tests/unit/components/labelPrinting/LabelPrintingForm.spec.js

```diff
@@ -0,0 +1,199 @@
+import LabelPrintingForm from '@/components/labelPrinting/LabelPrintingForm'
```

Comment:
It feels like things are being tested twice here. You have a thorough e2e test which fills in all of the fields and then they are being checked again here.